src/Utilities/_DataValidation.php

```diff
@@ -59,3 +59,41 @@ function validateOperationTimeout(?int $operationTimeout=null)
         }
     }
 }
+
+if (!function_exists('validateTruncateSize'))
```

Comment:
Interesting. Is this a PHP idiom?

Reply:
It's the pattern I found while looking into autoloaded functions. I assume the autoloader may try to load the file multiple times under some circumstances, and this check prevents a fatal error about functions already being defined.